runhouse/main.py

```diff
@@ -232,20 +343,43 @@ def status(
                 f"`runhouse ssh {cluster_name}` or `sky status -r` for on-demand clusters."
             )
             raise typer.Exit(1)
-        cluster_status = current_cluster.status()
     else:
-        current_cluster = rh.here
-        if not current_cluster or current_cluster == "file":
+        if not cluster_or_local or cluster_or_local == "file":
             console.print(
                 "\N{smiling face with horns} Runhouse Daemon is not running... \N{No Entry} \N{Runner}. "
                 "Start it with `runhouse restart` or specify a remote "
                 "cluster to poll with `runhouse status <cluster_name>`."
             )
             raise typer.Exit(1)
+
+    if cluster_or_local != "file":
         # If we are on the cluster load status directly from the object store
-        cluster_status: dict = obj_store.status()
+        cluster_status = obj_store.status()
+        cluster_config = copy.deepcopy(cluster_status.get("cluster_config"))
```

Comment:
a little confused about what's going on here, can we load the config from the object store instead of having to ping den? 
```cluster_config = obj_store.get_cluster_config()```